### PR TITLE
Add delete permissions modals and selection

### PR DESCRIPTION
**Pull request by fhlavac**
https://issues.redhat.com/browse/RHCLOUD-8351

- added removing permissions at the row kebab level, or at the toolbar kebab level
- enhanced tests

@mmenestr

### CODE REVIEW

File: src/smart-components/role/remove-permissions-modal.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RemoveModal from '../../presentational-components/shared/RemoveModal';
+
+const RemovePermissionsModal = ({ title, text, onClose, onSubmit, isOpen, confirmButtonLabel }) => {
+  return (
```
Comment (karelhala) [inline src/smart-components/role/remove-permissions-modal.js:6]: Redundant `return`

File: src/smart-components/role/remove-permissions-modal.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <RemoveModal
+      text={text}
+      title={title}
+      isOpen={isOpen}
+      confirmButtonLabel={confirmButtonLabel}
+      onClose={() => {
+        onClose();
+      }}
```
Comment (karelhala) [inline src/smart-components/role/remove-permissions-modal.js:14]: No need for closure
```suggestion
      onClose={onClose}
```

File: src/smart-components/role/remove-permissions-modal.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      onSubmit={() => {
+        onSubmit();
+      }}
```
Comment (karelhala) [inline src/smart-components/role/remove-permissions-modal.js:17]: Same as above
```suggestion
      onSubmit={onSubmit}}
```

**Comment by mmenestr** [comment]
Modal looks good! Just one small thing, can you change the action in the kebabs to say "Remove" instead of "Delete"?

**Comment by fhlavac** [comment]
> Modal looks good! Just one small thing, can you change the action in the kebabs to say "Remove" instead of "Delete"?

@mmenestr sure thing, missed that one, thanks :-)

**Comment by mmenestr** [comment]
Good to go!

**Review by karelhala — APPROVED** [review]

**Pull request merged**